default/python/AI/AIstate.py

```diff
@@ -364,8 +370,10 @@ def update_system_status(self):
         verbose = False
 
         # assess enemy fleets that may have been momentarily visible
-        cur_e_fighters = {CombatRatingsAI.default_ship_stats().get_stats(hashable=True): [0]}  # start with a dummy entry
-        old_e_fighters = {CombatRatingsAI.default_ship_stats().get_stats(hashable=True): [0]}  # start with a dummy entry
+        # start with a dummy entry
+        cur_e_fighters = {CombatRatingsAI.default_ship_stats().get_stats(hashable=True): [0]}
+        # start with a dummy entry
```

Comment:
Duplicate comment seems unneeded, go with "dummy entries" instead in the first comment.